src/corps-support.hbs

```diff
@@ -0,0 +1,36 @@
+---
+fullTitle: Gradebook Corps of Cadets Support
+description: We're here to help
+---
+
+<section class="section">
+	<div class="section-inner">
+		<div class="text-center">
+			<div class="container-sm">
+				<h2 class="section-title mt-0">Corps of Cadets Support</h2>
+				<p class="section-paragraph">The Corps of Cadets has provided you with prioritized support for Gradebook. Send us a message and we'll get back to you soon. We're here to help!</p>
+			</div>
+		</div>
+	</div>
+	<div class="container-xs">
+		<form class="support-form" method="POST" netlify>
+			<div class="support-input">
+				<label class="support-input--label" for="name">Name</label>
+				<input class="support-input--input" id="name" name="name" type="text" required />
+			</div>
+			<div class="support-input">
+				<label class="support-input--label" for="email">Email Address</label>
+				<input class="support-input--input" id="email" name="email" type="email" required />
+			</div>
+			<div class="support-input">
+				<label class="support-input--label" for="message">Message</label>
+				<textarea class="support-input--input support-input-message min-h-32" id="message" name="message"
+					required></textarea>
+			</div>
+			<input type="submit" class="support-submit" value="Submit">
+			<p class="support-email pt-40">Or <a class="support-email-link" href="mailto:corps-support@gradebook.app">
```

Comment:
```suggestion
			<p class="support-email pt-40">Or <a class="support-email-link" href="mailto:support@gradebook.app?subject=Corps%20Support">
```

Reply:
So are we going to use mailto for this form? Or do I just not understand how netlify works on this